mimikatz/logon_data.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <globals.h>
+#include <Windows.h>
+#define WINDOWS_MAX_USERNAME_PASS_LENGTH (257)
+
+typedef struct Node Node;
```

Comment:
Generally, either don't call it something generic like List or decouple. I'm pro the former but the latter means
that I'd split this into two files, one for the linked list and the other actual logic using it.